src/generators.cpp

```diff
@@ -13,7 +13,12 @@ namespace Generators {
 
 static bool _ = (Ort::InitApi(), false);
 
-OrtGlobals::OrtGlobals() : env_{OrtEnv::Create(OrtLoggingLevel::ORT_LOGGING_LEVEL_ERROR)} {}
+OrtGlobals::OrtGlobals()
+    : env_{OrtEnv::Create(OrtLoggingLevel::ORT_LOGGING_LEVEL_ERROR)},
+      arena_config_{OrtArenaCfg::Create(0, -1, -1, -1)} {
```

Comment:
>arena_config_ [](http://example.com/codeflow?start=6&length=13)

Do you need to hold the arena_config_ in OrtGlobals? Isn't a temporary variable enough?

Reply:
Thanks. Changed the code to use a local variable.